example/MODULE.bazel

```diff
@@ -27,7 +27,10 @@ use_repo(rules_ts_ext, "npm_typescript")
 # Configure and register the toolchain.
 llvm = use_extension("@toolchains_llvm//toolchain/extensions:llvm.bzl", "llvm")
 llvm.toolchain(
-    llvm_version = "16.0.0",
+    llvm_versions = {
+        "": "16.0.0",
+        "darwin-x86_64": "15.0.7",
```

Comment:
Why is the downgrade needed for macos? Why not downgrade for every os?

Reply:
they just don't publish LLVM for all platforms at each release, for some reason,

https://github.com/grailbio/bazel-toolchain/tree/master?tab=readme-ov-file#per-host-architecture-llvm-version